week-1/mandatory/2-classes-db/task.md

```diff
@@ -7,7 +7,28 @@ Below you will find a set of tasks for you to complete to consolidate and extend
 To submit this homework write the correct commands for each question here:
 
 ```sql
+1-  select * from rooms where rate > 100;
 
+2-  select * from reservations where checkin_date BETWEEN '2020-08-01' and '2020-09-01' and checkin_date + 3 < checkout_date;
```

Comment:
Very good - just a minor mistake, the BETWEEN operator gives and inclusive range, so it should be:
`... where checkin_date BETWEEN '2020-08-01' and '2020-08-30' ...`
You could also use:
`... WHERE date_trunc('month', checkin_date) = date_trunc('month', current_date) AND ...`